cli/args/mod.rs

```diff
@@ -1589,9 +1580,8 @@ impl CliOptions {
       || self.workspace().has_unstable("bare-node-builtins")
   }
 
-  pub fn unstable_detect_cjs(&self) -> bool {
-    self.flags.unstable_config.detect_cjs
-      || self.workspace().has_unstable("detect-cjs")
+  pub fn detect_cjs(&self) -> bool {
+    self.workspace().package_jsons().next().is_some()
```

Comment:
Is this correct? Doesn't it check for existence of `package.json` and not consulting if it has `type: "commonjs"`?

Reply:
That makes sense, consider adding a docstring here to avoid confusion in the future.